snake_n_ladders/lib/game.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+require_relative 'board'
+require_relative 'dice'
+require_relative 'user'
+require_relative 'game_configuration'
+
+class Game
+  attr_accessor :board, :status, :players, :dies, :configuration
+
+  def initialize
+    @status = :to_start
+    @dies = []
+    @players = []
+    @configuration = GameConfiguration.new(
+      end_game_rule: GameConfiguration::EndGameRule::FIRST_TO_FINISH,
+      invalid_dice_roll: GameConfiguration::InvalidDiceRoll::NONE,
+      extra_dice_roll: GameConfiguration::ExtraDiceRoll::ON_LENGTH
+    )
+  end
```

Comment:
setting defaults with for dice, board's length and breadth

Reply:
should we set default values in initializer?